src/udisksdaemonutil.h

```diff
@@ -96,6 +96,10 @@ gchar *udisks_daemon_util_escape_and_quote (const gchar *str);
 gchar *udisks_daemon_util_hexdump (gconstpointer data, gsize len);
 void udisks_daemon_util_hexdump_debug (gconstpointer data, gsize len);
 
+gchar *subst_str (const gchar *str, const gchar *from, const gchar *to);
+gchar *subst_str_and_escape (const gchar *str, const gchar *from, const gchar *to);
```

Comment:
I'm afraid these functions should have long stupid names if they are in this header file. I agree maybe these should be moved into something like *string_utils.c/h*.